ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp

```diff
@@ -755,11 +755,8 @@ void bind_binary_bw_optional(py::module& module, const binary_backward_operation
             * :attr:`output_tensor` (Optional[ttnn.Tensor]): preallocated output tensor
             * :attr:`queue_id` (Optional[uint8]): command queue id
 
-        Supported dtypes, layouts, and ranks:
-
-        {3}
-
-        Note : bfloat8_b/bfloat4_b is only supported on TILE_LAYOUT
+        Note:
+            {3}
```

Comment:
Update the description here in the format given in bind_binary_backward_overload